ceagle/api/client.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import requests
+
+from ceagle import config
+
+
+class Client(object):
```

Comment:
Client as instance looks nice to me because it at least keeps endpoint, and can be easily extended.

 # Using **client.Client** instance
 capacity = client.get_client("capacity")
 data1 = capacity.get("/some/path/1")
 data2 = capacity.get("/some/path/2")
 
 # Using some function **client.do_get()** instead
 data1 = client.do_get("capacity", "/some/path/1")
 data2 = client.do_get("capacity", "/some/path/2")